### PR TITLE
feat: allow setting child tags

### PR DESCRIPTION
**Pull request by jpbede**
## Description
The PR adds the ability to specify the allowed child tags of a custom component.

### CODE REVIEW

**Review by timo-reymann — APPROVED** [review]

**Comment by timo-reymann** [comment]
Heya @jpbede,

thanks for your contribution at first, nice feature! :)

Just a small thing: Please make the child tags attribute optional :)

File: src/CustomComponentOptions.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ export interface CustomComponentOptions {
      */
     allowedParentTags: string[]
 
+    /**
+     * Allowed tags for child
+     */
+    allowedChildTags: string[]
```
Comment (timo-reymann) [inline src/CustomComponentOptions.ts:25]: fix: `allowedChildTags: string[]` => `allowedChildTags?: string[]`

This way the attribute is not required and not a breaking change

File: test/MJMLCustComponent.test.ts
Comment (timo-reymann) [inline test/MJMLCustComponent.test.ts:1]: chore: Remove explicit `allowedChildTags`

File: test/integration.test.ts
Comment (timo-reymann) [inline test/integration.test.ts:1]: chore: Remove explicit `allowedChildTags`

**Review by timo-reymann — CHANGES_REQUESTED** [review]

**Comment by jpbede** [comment]
Reasonable, done :)

**Pull request merged**